tap_github/__init__.py

```diff
@@ -529,6 +529,7 @@ def getReposForOrg(org):
             # Preserve the case used for the org name originally
             namesplit = repo['full_name'].split('/')
             orgRepos.append(org + '/' + namesplit[1])
+            repo_cache[repo['full_name']] = repo
```

Comment:
Where is this defined? Do you need a global statement since you're modifying it? I presume this works -- maybe a global isn't needed when you're modifying a dict. My python isn't that great.

Reply:
It is an existing global that you had established elsewhere to store repo data of the same structure. A global statement is only needed if the variable will be reassigned via `=`, but when using it in another way, such as calling methods on it or accessing it via index/key, it is not needed.

Relevant SO question on this: https://stackoverflow.com/questions/14323817/global-dictionaries-dont-need-keyword-global-to-modify-them.

Reply:
Thanks, I'm not very familiar with python :-)